qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -123,6 +123,7 @@
         - others:
             variants:
                 - test_default_model:
+                    no RHEL.9
```

Comment:
So what's the mean of the default model not supported in RHEL9? `qemu64`?

Reply:
> So what's the mean of the default model not supported in RHEL9? `qemu64`?

Yes, it is deprecated, and RHEL9 guest will crash with this model.